moped-database/migrations/1650035696474_pk-and-sequences/up.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE moped_city_fiscal_years ADD COLUMN id integer;
+ALTER TABLE moped_city_fiscal_years DROP CONSTRAINT moped_city_fiscal_years_pkey;
+UPDATE moped_city_fiscal_years set id = fiscal_year_value::integer - 2015 where 1 = 1;
+ALTER TABLE moped_city_fiscal_years ALTER COLUMN id SET NOT NULL;
+CREATE SEQUENCE city_fiscal_years_id_seq AS INT INCREMENT BY 1 START WITH 7 OWNED BY moped_city_fiscal_years.id;
```

Comment:
just out of curiosity, why start with 7?

Reply:
It's a great question, they really do look like magic numbers in there setting the sequence like that. It's seven because there are six records in the table currently, so we want the next ID that gets doled out by the sequence to be 7.

Reply:
Thanks @chiaberry! Your comment and our discussion on slack revealed that I had chosen those next-ID values based on our seed data. I have looked at the read replica, and pushed a commit that updates the sequence creation for `moped_phases` to `14` for the next value.